src/types.ts

```diff
@@ -145,12 +152,25 @@ export interface PoolProtocolFeesGenerated {
     sevenDayProtocolFeesGeneratedInEth: number;
 }
 
+export interface RawAllTimeStakingStats {
+    total_rewards_paid: string;
+}
+export interface AllTimeStakingStats {
+    totalRewardsPaid: number;
```

Comment:
**NIT:** I realize that we're not consistent in other places but maybe we we should try to have the currency in the name for all amounts? Otherwise it's not entirely apparent if it's ZRX or ETH. (I will also update my PR to reflect this)
```suggestion
    totalRewardsPaidInEth: number;
```